keyboard/src/modules/Keyboard.rb

```diff
@@ -885,10 +885,18 @@ def Save
       )
       SCR.Write(path(".sysconfig.keyboard"), nil) # flush
 
-      cmd = "/usr/bin/localectl --no-convert set-keymap #{@keymap.gsub(/(.*)\.map\.gz/, '\1')}"
+      cmd = if Stage.initial
+        # lets hardcode /mnt there as we do not want to add dependency on yast2-installation
+        # and its Yast::Installation.destdir
+        "/usr/bin/systemd-firstboot --root /mnt --keymap '#{@keymap.gsub(/(.*)\.map\.gz/, '\1')}'"
```

Comment:
NP: I know, it was already there, but that `gsub` looks too complicated to me. What about simpler `@keymapgsub(/\.map\.gz/,"")` ?